Dockerfile

```diff
@@ -118,15 +119,6 @@ RUN mkdir /usr/local/src/mapserver/build && \
     make install && \
     ldconfig
 
-FROM pdok/lighttpd:1.4.65-buster-patch3 as service
```

Comment:
We moeten wel een multistage docker build blijven gebruiken, anders wordt de image te groot.